.github/workflows/Analyze.yml

```diff
@@ -0,0 +1,39 @@
+name: Analyze
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'java' ]
```

Comment:
Could you check swift, too? 😉

Reply:
you can use fastlane to build ios app (I remember command `bundle exec fastlane build` do build...)

Reply:
Sorry, slipped my mind 😢

Reply:
@RyuNen344 
Sorry for late... I encountered this error, and I'm not quite sure about it. 
It seems like it will take some time to investigate, so I'm thinking of merging the Java analysis first and then later submitting a PR for Swift.
Alternatively, do you have any possible causes or ideas that come to mind?
<img width="1220" alt="image" src="https://github.com/DroidKaigi/conference-app-2023/assets/39234775/b3923ce1-c1f7-470a-a09d-69abae4eef75">

Reply:
@tick-taku 

Really appreciate 😉 

>so I'm thinking of merging the Java analysis first and then later submitting a PR for Swift.

Ok, I gonna wait PR what you made!

>Alternatively, do you have any possible causes or ideas that come to mind?

hmm.... I've never seen these errors....

https://docs.github.com/ja/code-security/code-scanning/troubleshooting-code-scanning/no-source-code-seen-during-build

maybe I set `derived data path` to `root/app-ios/build`...?
could you check codeql without `derive data path` setting?
https://github.com/DroidKaigi/conference-app-2023/blob/main/app-ios/fastlane/Fastfile#L90

fyi: derive data path is temporary directory for build, like gradle project default output path